src/main/kotlin/me/zeroeightsix/kami/setting/SettingAnnotationProcessor.kt

```diff
@@ -50,7 +50,8 @@ object SettingAnnotationProcessor : LeafAnnotationProcessor<Setting> {
                             it.buildFuture()
                         }, "enum-${field.type.simpleName.toLowerCase()}"
                     )
-                    SettingInterface.Registry.add(interf.id, interf)
+                    //TODO: Don't just comment random stuff to make it compile!
+                    //SettingInterface.Registry.add(interf.id, interf)
```

Comment:
You forgot a TODO here :)

I fixed some things regarding registries, take a look at how it was done there.